src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(CreateFigures.values().length);
+        switch (CreateFigures.values()[index].name()) {
+            case "CIRCLE":
```

Comment:
```suggestion
        switch (CreateFigures.values()[index]) {
            case CreateFigures.CIRCLE:
```
You can use enum values for `switch case`